server/src/v1/tests/mentors.test.js

```diff
@@ -11,11 +11,15 @@ import generateToken from '../utils/authService';
 chai.use(chaiHttp);
 const { expect, request } = chai;
 
-describe('GET /api/v1/mentors', () => {
+describe.only('GET /api/v1/mentors', () => {
```

Comment:
I think you should start resolving tests with changing your endpoints

Reply:
I have removed the keyword 'only' all tests running now.